app/services/assignment_service.py

```diff
@@ -127,13 +126,15 @@ async def delete_assignment(self, assignment: AssignmentModel) -> None:
             FileOperation(content="", path=f"{ directory_path }", operation=FileOperationType.DELETE)
         ]
 
-        await gitea_service.modify_repository_files(
-            name=master_repository_name,
-            owner=owner,
-            branch_name=branch_name,
-            commit_message=f"Delete assignment",
-            files=files_to_modify
-        )
+        """ (maybe) this can come back in the future.
+        But it violates the merge control policy if active, which is problematic at the moment."""
+        # await gitea_service.modify_repository_files(
```

Comment:
we should probably comment out all the code above this as well. No reason it should run, but its not major